alf/optimizers/optimizers.py

```diff
@@ -170,16 +173,23 @@ def __init__(self,
                 kwargs["lr"] = float(lr())
         self._lr_schedulers = []
 
+        capacity_ratio = alf.utils.schedulers.as_scheduler(capacity_ratio)
+
+        # record initial capacity ratio in order to be used in ``add_param_group``,
+        # within which the capacity scheduler (``self._capacity_ratio``) is inaccessible
+        self._initial_capacity_ratio = capacity_ratio()
```

Comment:
why not assign self._capacity_ratio?

Reply:
Changed. I actually tried this initially when implementing this PR this yesterday, but was unsuccessful. Probably due to other issues at that time.  And that's why I recorded the initial capacity ratio value in the original PR.
Bow this seems unnecessary.